core/storage/base_model/gae_models.py

```diff
@@ -294,6 +293,7 @@ def get(
     def get(
         cls: Type[SELF_BASE_MODEL],
         entity_id: str,
+        *,
```

Comment:
Why are we adding * here and below? I'm not familiar with this syntax and it might need some explanation if we aren't using it elsewhere.

Reply:
Basically, we are using * to separate named arguments from keyword arguments. By this I mean if we need to use strict then we have to use it as a strict keyword argument if we use it like a normal named argument then MYPY will throw an error. So, to force developers to use strict as a keyword argument we used * here.

Currently, all the overloads that contain a strict argument, are following the * pattern. 
In this PR, these are the only places where * is not implemented.

Reply:
That's interesting. I feel like folks will not have any idea what this means and it's not straightforward to search for it. Could you perhaps add a comment at the top of each block of overloads explaining this? If you prefer you could also write a short blurb and point people to the relevant Mypy docs page.

Reply:
I think that as this is standard Python 3 syntax (although maybe not widely used), we shouldn't add comments for all occurrences of this (and if so then only some kind of blurb), I tried searching for "python 3 star as function argument" and the [first link](https://stackoverflow.com/questions/14301967/bare-asterisk-in-function-arguments) (at least for me) shows the explanation.

Should we maybe announce that we started using this syntax is some places through mailing list and just keep it without explanation?

Reply:
Ah, sorry, I thought this was some mypy thing and didn't realize it was standard Python 3.

In that case I agree with what you suggest. Thanks!

Reply:
Ok, I'll send an email to the oppia-dev once this PR gets merged.